flutter/shell/platform/tizen/tizen_view_base.h

```diff
@@ -35,6 +35,8 @@ class TizenViewBase {
   // Returns the geometry of the view.
   virtual TizenGeometry GetGeometry() = 0;
 
+  virtual uint32_t GetResourceId() = 0;
+
```

Comment:
Nothing should come between `GetGeometry` and `SetGeometry`. Please move `GetResourceId` to line 46 (after `GetDpi`).

Reply:
Fixed